### PR TITLE
Polishing the Superbowl card.

### PR DESCRIPTION
**Pull request by cyril-sf**
- "I'm ready" button in the playlist card
- Change intent of slot machine when playing/watching
- Hide spin button while spinning

### CODE REVIEW

**Comment by cyril-sf** [comment]
@hjdivad: good for review

File: example/cards/superbowl/card.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ var SlotMachineService = Conductor.Oasis.Service.extend({
   events: {
     getCoins: function() {
       card.adPlaylistCard.sandbox.adPlaylistPort.send('nextAd');
+      card.slotMachineCard.render('small', { width: 600, height: 200 });
```
Comment (hjdivad) [inline example/cards/superbowl/card.js:16]: Shouldn't we get these numbers from the superbowl card's dimensions?

File: example/cards/superbowl/card.js [hunk 2]
```diff
@@ -26,6 +27,10 @@ var AdPlaylistService = Conductor.Oasis.Service.extend({
     surveyTaken: function (data) {
       $('#slot_machine').show();
       card.slotMachineCard.sandbox.slotMachinePort.send('addCoin');
+    },
+    play: function(data) {
+      card.slotMachineCard.render('large', { width: 600, height: 200 });
```
Comment (hjdivad) [inline example/cards/superbowl/card.js:32]: Shouldn't we compute these numbers from the card's dimensions?

File: example/cards/video/card.js [hunk 3]
```diff
@@ -21,8 +21,8 @@ var card = Conductor.card({
   activate: function (data) {
     Conductor.Oasis.RSVP.EventTarget.mixin(this);
 
-    $('head').append('<script src="https://www.youtube.com/iframe_api"></script>');
     $('body').html('<img id="thumbnail" /><div id="player"></div>');
+    $('head').append('<script src="https://www.youtube.com/iframe_api"></script>');
```
Comment (hjdivad) [inline example/cards/video/card.js:25]: curious about the reordering.

**Comment by hjdivad** [comment]
@Cyril-sf seems good.  We have some hard coded dimensions.  Should we be computing them from the card's dimensions?

**Pull request merged**